CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 4.11.0 (2021-04-23)
+
+- Added the `startDateRequired` and `endDateRequired` input variables to the date range picker component. [#257](https://github.com/blackbaud/skyux-datetime/pull/257)
```

Comment:
Is `variables` correct? Or should this just be `inputs`?

Reply:
Updated.